src/pages/PrivacyStatement.tsx

```diff
@@ -13,28 +13,28 @@ export const PrivacyStatement = () => {
       <StickyLegalHeader text={t("privacy.header")} />
       <main className={styles.legalBackground}>
         <div className={styles.legalContainer}>
-          <LegalSection id="mIntro" header={t("privacy.intro.header")}>
+          <LegalSection id="intro" header={t("privacy.intro.header")}>
             <p> {t("privacy.intro.firstParagraph")} </p>
             <p> {t("privacy.intro.secondParagraph")} </p>
             <p> {t("privacy.intro.thridParagraph")} </p>
           </LegalSection>
           <LegalSection header={t("privacy.tableOfContents")}>
             <ul>
-              <li><a href="#mIntro"> {t("privacy.intro.header")} </a></li>
-              <li><a href="#mResponsible"> {t("privacy.responsibilities.header")} </a></li>
-              <li><a href="#mOverview"> {t("privacy.overviewOverProcesses.header")} </a></li>
-              <li><a href="#mLegalBases"> {t("privacy.relevantLegalBases.header")} </a></li>
-              <li><a href="#mSecurity"> {t("privacy.security.header")} </a></li>
-              <li><a href="#mDeletion"> {t("privacy.deletionOfData.header")} </a></li>
-              <li><a href="#mCookies"> {t("privacy.useOfCookies.header")} </a></li>
-              <li><a href="#mWebhosting"> {t("privacy.webhosting.header")} </a></li>
-              <li><a  href="#mVote"> {t("privacy.votes.header")} </a></li>
-              <li><a href="#mChanges"> {t("privacy.updates.header")} </a></li>
-              <li><a href="#mRights"> {t("privacy.rights.header")} </a></li>
-              <li><a href="#mDefinitions"> {t("privacy.definitions.header")} </a></li>
+              <li><a href="#intro"> {t("privacy.intro.header")} </a></li>
+              <li><a href="#responsible"> {t("privacy.responsibilities.header")} </a></li>
+              <li><a href="#overview"> {t("privacy.overviewOverProcesses.header")} </a></li>
+              <li><a href="#legal-bases"> {t("privacy.relevantLegalBases.header")} </a></li>
+              <li><a href="#security"> {t("privacy.security.header")} </a></li>
+              <li><a href="#deletion"> {t("privacy.deletionOfData.header")} </a></li>
+              <li><a href="#cookies"> {t("privacy.useOfCookies.header")} </a></li>
+              <li><a href="#webhosting"> {t("privacy.webhosting.header")} </a></li>
+              <li><a  href="#vote"> {t("privacy.votes.header")} </a></li>
```

Comment:
Is it just a visual error or are there two spaces behind the anchor tag?

Reply:
nah it isn't I fucked that up